core/storage/voiceover/gae_models.py

```diff
@@ -233,9 +233,11 @@ class VoiceArtistMetadataModel(base_models.BaseModel):
     # (strings) to lists of content IDs (strings), denoting the content IDs
     # for which voiceovers are provided in a given exploration by the
     # voice artist.
-    # - 'voiceovers': A list of sample voiceovers, where each voiceover is
+    # - 'exploration_id_to_voiceovers': A dict with exploration IDs as keys
+    # and the list of voiceovers as values, where each voiceover is
     # represented as a dictionary (VoiceoverDict). This field specifically
-    # contains the five sample voiceovers with the longest duration.
```

Comment:
Hi @seanlip wanted to highlight this, as I changed the plan to store all the voiceover information contributed by a user in a given exploration. I will fetch the 5 longest duration voiceovers in the controller layer. The reason for changing the plan is to simplify the code and test cases implementation i.e., removing 4-5 if-else clauses.
Thanks

Reply:
This seems reasonable to me -- thanks.

Reply:
Thank you for confirming